src/DiagramGenerator/Diagram/Board.php

```diff
@@ -520,12 +507,11 @@ protected function getCachedTextureFilePath()
     protected function getCachedPieceFilePath($pieceThemeName, $cellSize, $piece)
     {
         return sprintf(
-            '%s/%s/%d/%s.%s',
+            '%s/%s/%d/%s.png',
```

Comment:
yes, this was a mistake, because we were assuming the pieces and the board textures had the same image format, which was only true because all the board textures were in the png format. we were determining the image format from the boards, and that code would break now if we would remove the png board textures which we are currently still keeping. according to @rangogligoric the pieces will always be in the png format.